units/time_series_update.go

```diff
@@ -57,7 +58,7 @@ func NewUpdateTimeSeriesJob(series model.UnanalyzedPerformanceSeries) amboy.Job
 		series.Arguments,
 	)
 	j.SetID(fmt.Sprintf("%s.%s", baseID, time.Now().UTC()))
-	j.SetScopes([]string{baseID})
+	j.SetScopes([]string{baseID, strings.Join(series.Measurements, ",")})
```

Comment:
Is there any limit to how large this string can be? Joining all metrics would probably result in a very large string.

Reply:
FWIW it's already in an array in memory  (series.Measurements), so this is probably fine from a memory usage perspective.

Reply:
Yeah this is fine!